src/DurableTask.Netherite/OrchestrationService/PartitionErrorHandler.cs

```diff
@@ -61,7 +61,7 @@ public PartitionErrorHandler(int partitionId, ILogger logger, LogLevel logLevelL
             this.host = host;
         }
      
-        public void HandleError(string context, string message, Exception exception, bool terminatePartition, bool isWarning)
+        public void HandleError(string context, string message, Exception? exception, bool terminatePartition, bool isWarning)
```

Comment:
nit: Since we're making the `exception` parameter nullable here, I think we should do the same in the `TraceError` method.

Reply:
Indeed. Applied